packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.Props.ts

```diff
@@ -75,4 +75,12 @@ export interface ITooltipHostProps extends React.HTMLAttributes<HTMLDivElement |
    * Optional class name to apply to tooltip host.
    */
   hostClassName?: string;
+
+  /**
+   * Manual way to display tooltip when the element is not directly focusable but an ancestor is,
+   * so it can pass down that state via this property.
+   * It honors overflow rules and keyboard and mouse interaction, so this just adds another interaction way
+   * delegated on another element.
+   */
+  isFocused?: boolean;
```

Comment:
The problem I see with `forceShow` is that it doesn't mean that we're using the overflow logic, so `forceShow` true could result correctly in a hidden tooltip.
We're not forcing show unless there's overflow (in that mode), so I prefer seeing it from the other part: `simulateHasFocus` or `actLikeFocused` (both terrible names but I hope you get the point).

The larger discussion is whether this responsibility belongs to this control. IMHO it does, because we're just adding an external way to manage when tooltip visibility should be considered, in addition to own keyboard focus and mouse hovering that we had already in place. I agree though that naming should be improved.